main.go

```diff
@@ -35,7 +35,7 @@ import (
 
 	"github.com/ramendr/ramen/controllers"
 	rmnutil "github.com/ramendr/ramen/controllers/util"
-	// +kubebuilder:scaffold:imports
+	recipe "github.com/ramendr/recipe/api/v1alpha1"
```

Comment:
I think the `// +kubebuilder:scaffold:imports` ought to remain.  My earlier wondering was whether the recipe import should be before or after it.